Modules/Libs/QuestieLib.lua

```diff
@@ -535,3 +535,114 @@ function QuestieLib.tunpack(tbl)
 
     return recursion(1)
 end
+
+
+local frameObject = nil
+if _G["QuestLogObjectivesText"] then -- classic
+    frameObject = _G["QuestLogObjectivesText"] 
+elseif _G["QuestInfoObjectivesText"] then -- Wotlk Classic
+    frameObject = _G["QuestInfoObjectivesText"]
+end
+--Part of the GameTooltipWrapDescription function
+local objectiveFontString = UIParent:CreateFontString("questieObjectiveTextString", "ARTWORK", "QuestFont")
+objectiveFontString:SetWidth(frameObject:GetWidth() or 275) --QuestLogObjectivesText default width = 275
+objectiveFontString:SetHeight(0);
+objectiveFontString:SetPoint("LEFT");
+objectiveFontString:SetJustifyH("LEFT");
+---@diagnostic disable-next-line: redundant-parameter
+objectiveFontString:SetWordWrap(true)
+objectiveFontString:SetVertexColor(1,1,1, 1)--Set opacity to 0, even if it is shown it should be invisible
+local font, size = frameObject:GetFont()
+--Chinese? "Fonts\\ARKai_T.ttf"
+objectiveFontString:SetFont(font, size);
+objectiveFontString:Hide()
+
+---Emulates the wrapping of a quest description
+---@param line string @The line to wrap
+---@param prefix string @The prefix to add to the line
+---@param combineTrailing boolean @If the last line is only one word, combine it with previous? TRUE=COMBINE, FALSE=NOT COMBINE, default: true
+---@param splitOnDot boolean @Should we add a linebreak if a dot appears thats not at the end of a line TRUE=NEW ROW, FALSE=NO NEW ROW, default: true
+---@param desiredWidth number @Set the desired width to wrap, default: 275
+---@return table[] @A table of wrapped lines
+function QuestieLib:TextWrap(line, prefix, combineTrailing, splitOnDot, desiredWidth, questid)
+    if(objectiveFontString:IsVisible()) then print("TextWrap already running... strange, look into pooling the FontString ugh") end
+
+    --Set Defaults
+    combineTrailing = combineTrailing or true
+    splitOnDot = splitOnDot or true
+    --We show the fontstring and set the text to start the process
+    --We have to show it or else the functions won't work... But we set the opacity to 0 on creation
+    objectiveFontString:SetWidth(desiredWidth or frameObject:GetWidth() or 275) --QuestLogObjectivesText default width = 275
+    objectiveFontString:Show()
+
+    --Make a linebreak on each "dot" character if there is a space after (don't want it on end of line)
+    local useLine = string.gsub(line, "%. ", "%.%\n")
+
+    objectiveFontString:SetText(useLine)
+    --Is the line wrapped?
+    if(objectiveFontString:GetUnboundedStringWidth() > objectiveFontString:GetWrappedWidth()) then
```

Comment:
```suggestion
    local isLineWrapped = objectiveFontString:GetUnboundedStringWidth() > objectiveFontString:GetWrappedWidth()
    if(isLineWrapped) then
```

Reply:
Why do it this way? Isn't making a local variable with a descriptive name just a comment with more steps? :D Also worse performance because it saves it in memory!

Reply:
I come from a world where comments usually means a descriptive variable/method name is preffered 🤷🏻 But I am totally fine to keep it the way it is and I understand the performance aspect 😁 👍🏻